src/lib/eliom_client.client.ml

```diff
@@ -70,8 +70,13 @@ let get_global_data () =
     None
 
 let init_client_app
-    ~app_name ?(ssl = false) ~hostname ?(port = 80) ~full_path () =
+    ?app_path ~app_name
```

Comment:
`Eliom_request_info.get_csp_original_full_path` returns that field and describes it as "the full path of the URL where the client-side process is running". Is it what we need? I guess the client process can run on another URL, but not sure.